Forem for Safari/Forem for Safari/ViewController.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  ViewController.swift
+//  Forem for Safari
+//
+//  Created by Joshua Puetz on 1/15/21.
+//
+
+import Cocoa
+import SafariServices.SFSafariApplication
+import SafariServices.SFSafariExtensionManager
+
+let appName = "Forem for Safari"
+let extensionBundleIdentifier = "com.forem.Forem-for-Safari.Extension"
```

Comment:
> I can successfully build the app, but the button to 'launch safari' no longer works! Any ideas what could be causing that?

Did you also update this string to match the new lowercased identifiers?

Reply:
Ah! I sure didn't! Thanks, that worked. Note that I kept `Extension` capitalized as that's what the Xcode plugin conversion tool added!